### PR TITLE
 docs: fix changelog

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

**Review by henryiii — APPROVED** [review]
Just a thought; I'd be okay with not listing maintainers in the "thanks" sections, since it's implied that we all work on the package. I only added myself to the other couple of PRs because @jcfr added me to one already. Quite up to you, but just a thought, and I'm fine with being removed if that's cleaner.

**Comment by jcfr** [comment]
> with not listing maintainers in the "thanks" sections

Makes sense. I added commit removing @henryiii and @jcfr from the 0.12.0 entries.

> describing new scikit-buiild organization maintainer

@henryiii  and @mayeut:  Let me know what you think of this last commit I pushed where I tried to describe your recent contribution.

@henryiii  It may also be worth referencing the fact your presented sickit-build at SciPy and linking to the slides ? What do you think ?

cc: @thewtex

File: CHANGES.rst [hunk 1]
```diff
@@ -8,6 +8,19 @@ details, see the commit logs at http://github.com/scikit-build/scikit-build
 Scikit-build 0.12.0
 ===================
 
+The scikit-build GitHub organization welcomes :user:`henryiii` and :user:`mayeut` as core contributors
+and maintainers.
+
+:user:`henryiii` has been instrumental to add Apple Silicon support, add support for Visual Studio 2019, update
+the Continuous Integration infrastructure as well as helping review & integrate contributions, and address
+miscellaneous issues. Additionally, :user:`henryiii` worked on adding `example project <https://github.com/pybind/scikit_build_example>`_
+to built with ``pybind11`` and ``scikit-build``.
```
Comment (henryiii) [inline CHANGES.rst:17]:
```suggestion
:user:`henryiii` is a `pybind11 <https://pybind11.readthedocs.io>`_ and `pypa/build <https://pypa-build.readthedocs.io>`_ maintainer, has been instrumental in adding Apple Silicon support, adding support for Visual Studio 2019, updating
the Continuous Integration infrastructure, as well as helping review & integrate contributions, and addressing
miscellaneous issues. Additionally, :user:`henryiii` has worked on an `example project <https://github.com/pybind/scikit_build_example>`_  to build with ``pybind11`` and ``scikit-build``.
```

File: CHANGES.rst [hunk 2]
```diff
@@ -8,6 +8,19 @@ details, see the commit logs at http://github.com/scikit-build/scikit-build
 Scikit-build 0.12.0
 ===================
 
+The scikit-build GitHub organization welcomes :user:`henryiii` and :user:`mayeut` as core contributors
+and maintainers.
```
Comment (henryiii) [inline CHANGES.rst:12]:
```suggestion
The scikit-build GitHub organization welcomes :user:`henryiii` and :user:`mayeut` as core contributors
and maintainers. Both are also maintainers of `cibuildwheel <https://cibuildwheel.readthedocs.io>`_.
```

File: CHANGES.rst [hunk 3]
```diff
@@ -8,6 +8,19 @@ details, see the commit logs at http://github.com/scikit-build/scikit-build
...
+:user:`henryiii` has been instrumental to add Apple Silicon support, add support for Visual Studio 2019, update
+the Continuous Integration infrastructure as well as helping review & integrate contributions, and address
+miscellaneous issues. Additionally, :user:`henryiii` worked on adding `example project <https://github.com/pybind/scikit_build_example>`_
+to built with ``pybind11`` and ``scikit-build``.
+
+:user:`mayeut` is the maintainer of `cibuildwheel <https://cibuildwheel.readthedocs.io>`_ and
+focused his effort on updating the ``cmake-python-distributions`` and ``ninja-python-distributions`` so
+that the corresponding wheels are available on all supported platforms including Apple Silicon and all flavors
+of manylinux.
+
```
Comment (henryiii) [inline CHANGES.rst:23]:
```suggestion
:user:`mayeut` is a `manylinux <https://github.com/pypa/manylinux>`_ maintainer and
focused his effort on updating the ``cmake-python-distributions`` and ``ninja-python-distributions`` so
that the corresponding wheels are available on all supported platforms including Apple Silicon and all flavors
of manylinux.

```

@mayeut is out for a couple of weeks, but I think this is a bit better, if you want to list related projects we are maintainers of.

**Review by henryiii — COMMENTED** [review]
Here are some suggestions (GitHub's suggestion tool is misbehaving currently, but I think these are showing up correctly).

**Comment by jcfr** [comment]
@henryiii  Thanks for the review :pray:

**Pull request merged**